easybuild/framework/easyconfig/default.py

```diff
@@ -155,12 +155,13 @@
     'exts_list': [[], 'List with extensions added to the base installation', EXTENSIONS],
 
     # MODULES easyconfig parameters
+    'modaliases': [{}, "Aliases to be defined in module file", MODULES],
     'modextrapaths': [{}, "Extra paths to be prepended in module file", MODULES],
     'modextravars': [{}, "Extra environment variables to be added to module file", MODULES],
     'modloadmsg': [{}, "Message that should be printed when generated module is loaded", MODULES],
     'modluafooter': ["", "Footer to include in generated module file (Lua syntax)", MODULES],
+    'modname': [None, "Module name to use (rather than using software name", MODULES],
```

Comment:
not very happy with this parameter name, since it's not clear this only covers part of the full module name (only the bit that is now covered by `name`)

maybe it should be `name_in_module`?

@rjeschmi, @wpoely86: suggestions?